modules/install-sno-ibm-z-lpar.adoc

```diff
@@ -0,0 +1,188 @@
++
+[IMPORTANT]
+====
+The {op-system} images might not change with every release of {product-title}. You must download images with the highest version that is less than or equal to the {product-title} version that you install. Only use the appropriate `kernel`, `initramfs`, and `rootfs` artifacts described in the following procedure.
+====
++
+The file names contain the {product-title} version number. They resemble the following examples:
++
+`kernel`:: `rhcos-<version>-live-kernel-<architecture>`
+`initramfs`:: `rhcos-<version>-live-initramfs.<architecture>.img`
+`rootfs`:: `rhcos-<version>-live-rootfs.<architecture>.img`
++
+[NOTE]
+====
+The `rootfs` image is the same for FCP and DASD.
+====
+
+. Move the following artifacts and files to an HTTP or HTTPS server:
+
+** Downloaded {op-system-base} live `kernel`, `initramfs`, and `rootfs` artifacts
+** Ignition files
+
+. Create parameter files for a particular virtual machine:
++
+.Example parameter file - need input!!!
++
+[source,terminal]
+----
+rd.neednet=1 \
+console=ttysclp0 \
+coreos.live.rootfs_url=<rhcos_liveos>:8080/rootfs.img \// <1>
+ignition.firstboot ignition.platform.id=metal \
+ignition.config.url=<rhcos_ign>:8080/ignition/bootstrap-in-place-for-live-iso.ign \// <2>
+ip=encbdd0:dhcp::02:00:00:02:34:02 <3>
+rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
+rd.dasd=0.0.4411 \// <4>
+rd.zfcp=0.0.8001,0x50050763040051e3,0x4000406300000000 \// <5>
+zfcp.allow_lun_scan=0 \
+rd.luks.options=discard
+----
+<1> For the `coreos.live.rootfs_url=` artifact, specify the matching `rootfs` artifact for the `kernel`and `initramfs` you are booting. Only HTTP and HTTPS protocols are supported.
+<2> For the `ignition.config.url=` parameter, specify the Ignition file for the machine role. Only HTTP and HTTPS protocols are supported.
+<3> For the `ip=` parameter, assign the IP address automatically using DHCP or manually as described in "Installing a cluster with z/VM on {ibm-z-name} and {ibm-linuxone-name}".
+<4> For installations on DASD-type disks, use `rd.dasd=` to specify the DASD where {op-system} is to be installed. Omit this entry for FCP-type disks.
+<5> For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed. Omit this entry for DASD-type disks.
++
+Leave all other parameters unchanged.
+
+. Transfer the following artifacts, files, and images to z/VM. For example by using FTP:
+
+** `kernel` and `initramfs` artifacts
+** Parameter files
+** {op-system} images
++
+For details about how to transfer the files with FTP and boot, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_8_installation/installing-in-an-lpar_installing-rhel[Installing in an LPAR].
+
+. Boot the machine
```

Comment:
Since this is a process with bootstrap separate, we have to boot both the bootstrap and then subsequently the SNO LPAR (or the master) 

Consider this line here instead. 

```
Boot the bootstrap and master machine with their respective ignition config and boot params.
```

Reply:
Discussed with @smolin-de that "Boot the machine" is sufficient.